avocado/utils.py

```diff
@@ -0,0 +1,176 @@
+from django_rq import get_queue
+
+from avocado.conf import settings
+from avocado.export import HTMLExporter, registry as exporters
+from avocado.query import pipeline
+import avocado.query.utils as query_utils
+
+
+def get_exporter_class(export_type):
+    if export_type.lower() == 'html':
+        return HTMLExporter
+
+    return exporters[export_type]
+
+
+def async_get_result_rows(context, view, query_options, job_options=None):
+    """
+    Creates a new job to asynchronously get result rows and returns the job ID.
+
+    Args:
+        See get_result_rows argument list.
+
+    Keyword Arugments:
+        Set as properties on the returned job's meta.
+
+    Returns:
+        The ID of the created job.
+    """
+    if not job_options:
+        job_options = {}
+
+    queue = get_queue(settings.ASYNC_QUEUE)
+    job = queue.enqueue(
+        get_result_rows, context, view, query_options, evaluate_rows=True)
```

Comment:
Nit. Can you change anywhere you need to wrap arguments one per line? A bit easier on the eyes and on diffs.

```python
job = queue.enqueue(get_result_rows,
                    context,
                    view,
                    ...)
```